resources/projects/libraries/generic_robot_window/generic.c

```diff
@@ -0,0 +1,278 @@
+/*
+ * Copyright 1996-2021 Cyberbotics Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <webots/plugins/robot_window/default.h>
+#include <webots/plugins/robot_window/generic_robot_window/generic.h>
```

Comment:
`generic_robot_window` is a separate library. 
I think that this extra layer is very important to make it more clear that it is a different library than the libController. Saving header files from different libraries in the same folder seems a bad practice.

Reply:
It refers to this robot window: `$WEBOTS_HOME/resources/projects/plugins/robot_windows/generic/`, so I would at least rename the folder so that the include file is `$WEBOTS_HOME/include/controllers/c/webots/plugins/robot_window/generic/generic.h` which seems more consistent, isn't it?

Reply:
No, it refers to `$WEBOTS_HOME/resources/projects/libraries/generic_robot_window/`.

Reply:
OK, I see, it doesn't refer to a robot window, but to a library for building robot windows...
So, let's keep it like that.